### PR TITLE
Add CPython urlparse test

### PR DESCRIPTION
**Pull request by hyperbora**
test_urlparse.py

### CODE REVIEW

File: Lib/test/test_urlparse.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def test_qsl(self):
             expect_without_blanks = [v for v in expect if len(v[1])]
             result = urllib.parse.parse_qsl(orig, keep_blank_values=False)
             self.assertEqual(result, expect_without_blanks,
-                            "Error parsing %r" % orig)
+                             "Error parsing %r" % orig)
```
Comment (youknowone) [inline Lib/test/test_urlparse.py:143]: There are simple format changes not related to the unsupported test marking. Will you please remvoe these changes?

**Comment by hyperbora** [comment]
OK, I will edit some changes, and send PR later

**Comment by youknowone** [comment]
@hyperbora You don't need to close and open another PR to fix it. Edit the last commit from your local enviroment and push(force-push) it to your `more-tests` branch (which you used for this PR). Then this PR will be updated to the new contents.

**Comment by hyperbora** [comment]
> @hyperbora You don't need to close and open another PR to fix it. Edit the last commit from your local enviroment and push(force-push) it to your `more-tests` branch (which you used for this PR). Then this PR will be updated to the new contents.

Edit the last commit from my "more-tests" branch

**Comment by youknowone** [comment]
Thanks for the work. Now the result looks fine. Because we don't need intermediate works including revert here, would you please also rebase the work? `git rebase -i HEAD^^^^` will give you interactive editor with 4 recent commits. Remove 2 intermediate commits and then push remaining 2 commits again please. `push --force` is intended to be used here so please don't afraid of it.

**Comment by hyperbora** [comment]
Thank you for your comment. i fixed my branch. :)

**Pull request merged**